lambda/custom/helpers/dbHelper.js

```diff
@@ -1,17 +1,17 @@
 var AWS = require("aws-sdk");
 AWS.config.update({region: "us-east-1"});
-const tableName = "cool-actor-database";
+const tableName = "flashytable"; //change to your Dynamodb table name
 
 var dbHelper = function () { };
 var docClient = new AWS.DynamoDB.DocumentClient();
 
-dbHelper.prototype.addActor = (movie, actor, userID) => {
+dbHelper.prototype.addCard = (answer, card, userID) => {
     return new Promise((resolve, reject) => {
         const params = {
             TableName: tableName,
             Item: {
-              'movieName' : movie,
-              'actorName' : actor,
+              'back' : answer,
+              'front' : card,
```

Comment:
food for thought on this one--how are we writing table?
Mine is like this:
Item: {
              'cardTitle' : actor, //these are how they correspond to the starter code
              'cardContent' : movie,// would change to 'term' or 'topic' and 'description' respectively.
              'userId': userID
            }
//Other thought is, not directly realted to this line, these are not plug and play because of that db aspect.🤔